cookbook/entities/acl.rst

```diff
@@ -0,0 +1,246 @@
+How to Protect Resources Using ACLs
+===================================
+
+Using ACLs you can granularly grant access to your entities. Doing so requires three steps:
+
+#. :ref:`Activate ACL checks for your entities <coobook-entities-acl-enable>`.
+
+#. :ref:`Create access control lists for all available actions <coobook-entities-acl-create>`.
+
+#. :ref:`Add access checks<coobook-entities-acl-check>` to where your entities are displayed or
+   manipulated.
+
+.. _coobook-entities-acl-enable:
+
+Activating ACL Checks on your Entities
+--------------------------------------
+
+In order to have your entity available in the admin UI to be able to assign permissions to your
+users you have to enable ACLs for these entities using the ``@Config`` annotation:
+
+.. code-block:: php
+    :linenos:
+
+    // src/AppBundle/Entity/Task.php
+    namespace AppBundle\Entitiy;
+
+    use Oro\Bundle\EntityConfigBundle\Metadata\Annotation\Config;
+
+    /**
+     * @Config(
+     *     defaultValues={
+     *         "security"={
+     *             "type"="acl",
+     *             "group_name"=""
+     *         }
+     *     }
+     * )
+     */
+    class Task
+    {
+        // ...
+    }
+
+After you have done this and have cleared the cache you can toggle all kinds of permission checks
+(``CREATE``, ``EDIT``, ``DELETE``, ``VIEW``, and ``ASSIGN`` in the user role management interface.
+
+.. tip::
+
+    You can use the optional group attributes to create visual groups in the UI that can help the
```

Comment:
Now group have another behavior (from https://github.com/orocrm/platform/blob/master/src/Oro/Bundle/SecurityBundle/Resources/doc/implementation.md):

`group_name` parameter is used to group entities by applications. It is used to split security into application scopes.